### ISSUE TITLE
Cannot connect to private repositories

### ISSUE THREAD

**Issue by OPeyrusse** [mvndaemon/mvnd#415]
My maven configuration contains private repositories, to which I connect passing credentials as follow:

```xml
<settings>
	...
	<repository>
		<id>some-id</id>
		...
	</repository>
	<server>
		<id>some-id</id>
		<username>...</username>
		<password>...</password>
	</server>
</settings>
```

With such a configuration, I regularly see 403 errors when trying to access artifacts only available on those repositories.
When using a standard maven, I don't have any issue.

I would gladly try to reproduce and share but I don't know any private repo with open credentials, and I don't want to share my own :)

**Comment by gnodet** [comment]
Maybe using some docker image based on Apache Archiva could help setting up an integration test / reproducer ?

**Comment by OPeyrusse** [comment]
> Maybe using some docker image based on Apache Archiva could help setting up an integration test / reproducer ?

Good idea. I was not aware of this project (but I confess I did not make a lot of research). I will try to set up something and report here if I see an issue.

**Comment by lihao0324** [comment]
> Maybe using some docker image based on Apache Archiva could help setting up an integration test / reproducer ?

This is a serious bug, half of the jars in our project are in self-service, this will cause the build to fail, please look into this issue, thank you.

**Comment by grubeninspekteur** [comment]
I'm pointing to some custom Maven Settings with `maven.settings` in `mvnd.properties`. It picks up the correct repositories, but it does not seem to pick up the settings-security.xml next to it, at least I get 403 errors as well.

**Comment by OPeyrusse** [comment]
I completely forget about this issue and I don't use this feature anymore. @grubeninspekteur , feel free to try reproducing it on your side. As much as I am concerned, I could close this ticket as stale
